.github/workflows/ci.yml

```
@@ -0,0 +1,45 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+        runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Fetch configlet
+        uses: exercism/github-actions/configlet-ci@master
+
+      - name: Configlet Linter
+        run: configlet lint --track-id pharo .
+    
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Fetch smalltalkCI
+        uses: hpi-swa/setup-smalltalkCI@v1
```

It's best practice to target specific commit hashes over tags: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions
```suggestion
        uses: hpi-swa/setup-smalltalkCI@a271ee530f18ecf947856f3f163c4687101ec6b6
```

---

Exceptions to this are: official actions (`actions/*`) and those you control (`exercism/github-actions/*`)